python_modules/dagster/dagster/_core/definitions/auto_materialize_rule.py

```diff
@@ -751,6 +895,117 @@ def from_rule_evaluation_results(
                 rule_snapshots=auto_materialize_policy.rule_snapshots,
             )
 
+    def _deserialize_result(
+        self,
+        serialized_result: Tuple[
```

Comment:
This name is confusing to me, because I would expect an object named `serialized_x` to be a `str` or byte buffer.

Would it make sense to have separate `rule_evaluation` and `serialized_partitions_subset` arguments?